.travis.yml

```diff
@@ -36,8 +36,6 @@ install:
 before_script:
   - echo "zend_extension=xdebug.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
 
-script:
-  - vendor/bin/phpunit -c phpunit.xml.dist --coverage-clover build/logs/clover.xml
+script: ./vendor/bin/phpunit tests/YahooWeatherClientTests.php --coverage-html --coverage-clover=coverage.xml
```

Comment:
this line can be reduced to 
```
- vendor/bin/phpunit -c phpunit.xml
```
Because all the configuration is now in the XML, but i don't know if `-c` option is always necessary